### DISCUSSION TITLE
zoom

### DISCUSSION THREAD

**Discussion by ichnilatis-gr** [koreader/koreader#7084] [Q&A] [answered]
Is there any possibility for a smoother zoom in/out in pdf, djvu, pics etc and the reflowable documents (i.e. increase/decrease letters) with pinch and spread gestures?

Actually, when I make these gestures there is no immediate response or the letters increase while I pinch and vise versa, even I have set pinch to "decrease font size" and spread to "increase font size" (reflowable documents) and to "zoom manual" for pdf, djvu etc.

I believe that you understand what I mean... The response that we have with all touch screen devices...
Is there any setting I miss?

Thanks!

**Comment by Frenzie** [comment, marked as answer]
Not right now. The program is designed for devices where the experience is significantly better with a few standard zoom modes like "fit to X". It'd be a nice addition, but there are already plenty of document viewers out there that should allow you to do that.

Reflow is far too demanding (on ereaders and possibly even desktop) to be immediate. It'll take a second.
